Index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE >
+<html lang="en">
+
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+
+  <title>Protfolio Mobile-setup</title>
+
+  <!-- Load Styles' file -->
+  <link href="styles.css" rel="stylesheet">
+
+  <!-- Load Fonts -->
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+  <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&display=swap" rel="stylesheet">
+</head>
+
+<body>
+  <header class="header_section">
+
+    <img src="Sources/Union.png" alt="Unicon vector" id="unicon">
+    <span id="name">Omar Farouq</span>
+
+  </header>
+  <main>
+
+    
+
+    <section class="headline_section">
+
+        <img src="Sources/Group.png" alt="illustrations' vector"
+         id="bkg_illustrations">
+        
+        <div id="primary_text">
+          Hey There. I’m Omar
+        </div>
+
+        <div id="supporting_text">
+            I’am a Software Developer
+        </div>
```

Comment:
Please consider wrapping this part of the headline in a `p` tag instead of the generic `div` tag. For the page to be semantically correct it would not be best practice to follow an `h1`  tag with a description of what the page is about thus a `p` tag would be the most suitable tag.